packages/kotti-ui/source/kotti-navbar/KtNavbar.vue

```diff
@@ -252,7 +273,6 @@ $narrow-navbar-width: 3.4rem;
 	.kt-navbar-quick-links {
```

Comment:
do u think following BEM since we now have `kt-navbar` this can become `kt-navbar__quick-links`
previously we didn't consider probably bc we didn't have it

assuming we don't need to overwrite it in Yoda this should be fine but this will be a breaking change and we can adjust release notes accordingly to handle this on any user app's side 

and then also move within the above class as `&__quicklinks`

Reply:
i went through the file and it seems the entire BEM convention is absolutely weird for this file, subcomponents seem to be separated with `-` in many cases

you can skip this comment, and i'll make time for a separate PR when u merge yours

sorry for the hassle

Reply:
Yes this is old and not really using BEM. 
Ok for separate PR :+1: